.github/workflows/build-ci.yml

```diff
@@ -0,0 +1,80 @@
+name: build-ci
+
+on:
+  pull_request:
+    paths:
+      - "**/Dockerfile"
+      - "**/docker-entrypoint.py"
+      - "**/PLATFORMS"
+      - "tests/"
+      - "tools/genmatrix.js"
+      - ".github/workflows/build-ci.yml"
+
```

Comment:
For master only ? While we don't have a branch strategy
[warning indentation, to copy manually if used]
```suggestion
      - ".github/workflows/build-ci.yml"
    branches:
      - master         
```

Reply:
Yes, not necessary a good idea